omod/src/main/web/dashboardwidgets/latestobsforconceptlist/latestobsforconceptlist.controller.js

```diff
@@ -7,11 +7,12 @@ export default class LatestObsForConceptListController {
 
     $onInit() {
         this.maxConceptCount = 10;
-        this.maxAgeInDays = undefined;
+        this.maxAgeInDays = void 0;
```

Comment:
what's this?

Reply:
Hi @kaweesi, actually, no problem at all.  I changed it from `undefined` to `void 0` due to recommendations from the Codacy PR review.